firmware/watchguard_sysa-dl.ksy

```diff
@@ -0,0 +1,139 @@
+meta:
+   id: wfsysadl
+   endian: be
+   title: Watchguard Fireware web update (Sysa-dl) file format
+   license: MIT
+
+
+seq:
+  - id: fwheader
+    type: fwheader
+    size: 24
+
+  - id: sections
+    type: sections
+    size: _io.size - 24
```

Comment:
This is best replaced by this:

```
size: fwheader.file_size
```

This is preferred to tying it to the size of the whole file: in case there is extra data appended to the file the above will not file, while your method would fail.